src/test/java/seedu/address/model/student/StudentTest.java

```diff
@@ -1,79 +1,5 @@
 package seedu.address.model.student;
 
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_ADDRESS_BOB;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_EMAIL_BOB;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_BOB;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_PHONE_BOB;
-import static seedu.address.testutil.TypicalStudents.ALICE;
-import static seedu.address.testutil.TypicalStudents.BOB;
-
-import org.junit.jupiter.api.Test;
-
-import seedu.address.testutil.StudentBuilder;
-
 public class StudentTest {
 
-    @Test
-    public void isSameStudent() {
-        // same object -> returns true
-        assertTrue(ALICE.isSameStudent(ALICE));
-
-        // null -> returns false
-        assertFalse(ALICE.isSameStudent(null));
```

Comment:
TuitionTest replaces StudentTest. What tests should we have here now?

Reply:
Will need to write test cases for Students!